monitoring-as-code/README.md

```diff
@@ -34,17 +42,45 @@ docker build -t sre-monitoring-as-code:latest .
 
 ## Useage
 
-```
-# Add mixin file <service>-mixin.jsonnet to /montoring-config
-touch grapi-mixin.jsonnet
+### Default mixin config 
 
-# Add Global and SLI configuration as per sre-monitoring-as-code docs (see Resources)
+**To run the default monitoring and summary mixins bundles into the built container run the following command**
 
+```
 # Execute makefile script
 sh deploy.sh
+```
+
+### Custom mixin
+
+**To run a custom mixin file**
 
-# Add artefacts (dashboard, alerts rules and recording rules) to Grafana and Prometheus package management tooling (Prometheus Operator)
 ```
+# Add mixin file <service>-mixin.jsonnet to a directory
+touch grapi-mixin.jsonnet
+
+# Execute docker run command based on mounted directory where the mixin file has been added.
+docker run --mount type=bind,source="$PWD"/{user input directory},target=/input --mount type=bind,source="$PWD"/{user output director},target=/output -it sre-monitoring-as-code:{tag} -m {service} -rd -i input -o output
```

Comment:
Typo with user output director, should be user output directory

Reply:
Corrected typo. Thanks.